forge/routes/auth/index.js

```diff
@@ -399,14 +399,14 @@ module.exports = fp(async function (app, opts, done) {
                 // invite.inviteeId = verifiedUser.id
                 // await invite.save()
             }
-            await userLog(request.session.User.id, 'verify.verify-token', {
+            await userLog(request.session?.User?.id || verifiedUser.id, 'verify.verify-token', {
                 user: {
                     username: verifiedUser.username,
                     name: verifiedUser.name,
                     email: verifiedUser.email,
                     admin: !!verifiedUser.isAdmin
                 }
-            })
+            }, verifiedUser?.id)
```

Comment:
Not sure we need this (especially as we've used it unguarded in the 2 other changed lines
```suggestion
            }, verifiedUser.id)
```

Reply:
yup - you are right